index.js

```diff
@@ -4,11 +4,79 @@ const graphqlHTTP = require('express-graphql');
 const cors = require('cors');
 const root = require('./src/resolvers/RootResolver');
 const schema = require('./src/schema/schema');
+const GitHubStrategy = require('passport-github2').Strategy;
+const passport = require('passport');
+const partials = require('express-partials');
+// const util = require('util');
+const session = require('express-session');
+const methodOverride = require('method-override');
```

Comment:
Not needed - method override allows you use HTTP verbs such as PUT or DELETE in places where the client doesn't support it. We aren't using PUT or DELETE